### DISCUSSION TITLE
how to use local Executors in 13.10.0

### DISCUSSION THREAD

**Discussion by G233** [nrwl/nx#9746] [Show and Tell]
how to use local Executors in 13.10.0

**Comment by gioragutt** [comment]
To use a local executor in >=13.10, you can create a local plugin using `nx g @nrwl/nx-plugin:plugin`.

Say your scope is `@my-company` and you named your plugin `awesome-plugin`.

You can now generate an executor using `nx g @nrwl/nx-plugin:executor --project=awesome-plugin`, and let's say you name it `"build"`.

In a `project.json`, you'd use it like so:

```json
{
  "targets": {
    "build": {
      "executor": "@my-company/awesome-plugin:build",
      "options": {}
    }
  }
}
```

Does that answer your question?
